products/virtual-agent/research/inclusive-design-interviews/inclusive-design-interviews-conversation-guide.md

```diff
@@ -18,39 +18,139 @@
+   - I may need to pause you at times an ask you what you're hearing if I can't make out what's being read.
+
+**For those joined by iPhone and Zoom already**
+SKIP TO [Screen sharing](#screen-sharing) SET UP
+
+## Joining Zoom
+
+### Help participant join with device
+
+- I'm going to send you a zoom link [by format specified] for you to open on your phone so you can join this meeting.  
+- [Send through Veteran's preferred communication channel]
+- When you open the link, you should be taken to the Zoom application automatically. Zoom will ask for access to your microphone and video. We only need your audio for this session, so please tap "Allow" for the microphone prompt. Feel free to tap "Don't Allow" for video permissions if you are not comfortable having your face on camera.
+- Upon joining the room, Zoom will have you turn on audio by asking “To hear others, please join audio.” Please swipe to and double tap "Wifi or Cellular Data." [This button will be at the center of their screen.]
+- Once you're in the call...
+
+## Screen sharing
+
+At this point [I'd like you to go back to Zoom] I'm going to have you share your Zoom screen so that I can follow along with you. 
+
+### VoiceOver iOS (Mobile and Tablet) using web app
+
+ - Have you shared your screen before on Zoom?
+[If no] Please open your iPhone settings. Once you’re there, go to your control center. Then, find the “Insert screen recording” button and double tap to activate it. When you’re done, please return back to the zoom app.
+ - Within Zoom, I'd like you to find the "Share content" button and double tap it. You may have to swipe a number of times.
+ - A menu should appear. Select the "Screen” option and double tap it. 
+   - If you haven’t given Zoom permission to record yet on your device, you’ll be prompted to give access to allow it”
+ - Next you'll want to swipe to the "Start Broadcast" button and double tap that. You'll hear a counter "3, 2, 1" and the screen share should begin.
+ - Please swipe right until you hear the option to share your audio.
+ - [If the screen is black]: We see a black screen right now. If you have screen curtain or other screen hiding tech running, please toggle it off if possible.
+ - [If you don’t hear the screen reader]: We can’t hear your screen reader. Are you using headphones, hearing aids, or other technology? If so, would you be able to comfortably use your phone/desktop without that device so we can hear your screen reader? If not, that’s perfectly OK.
+ - Return to your home screen and the screen share should begin.
+
+[If successful:]
+
+OK, great, I can see your screen. Sometimes Zoom can be finicky and I will lose the screen share connection and I may have to ask you share screen again the way we just did. Also, if you need to stop sharing during this session you can double tap “Stop Share.” It’s located where the “Screen Share” button originally was at the bottom center of your screen.
```

Comment:
Let's keep watch on the last sentence here, that uses location "bottom center of your screen" to identify the "Screen Share" button. It should be fine for mobile users because they have a physical device to touch and establish a frame of reference where the screen bottom is, but desktop users will not. I don't have guidance to change it, but just something to note.